### ISSUE TITLE
Guess $SVN_USERNAME if not provided as secret and make it optional

### ISSUE THREAD

**Issue by evrpress** [10up/action-wordpress-plugin-asset-update#41]
**Is your enhancement related to a problem? Please describe.**

The username `$SVN_USERNAME` can normally be found in the list of contributors in the readme. If `$SVN_USERNAME` is not defined use the very first one in from the list of contributes.

This will help if you define `$SVN_PASSWORD` in your organizations secrets and you don't have to redo it for every repo

**Describe the solution you'd like**

Check if the `$SVN_USERNAME` is defined and set it if not.

```bash
if [[ -z "$SVN_USERNAME" ]]; then
	echo "Set the SVN_USERNAME secret"
	SVN_USERNAME=`grep  "^Contributors:[^\s]*" $README_NAME | awk '$1=="Contributors:"{print $2}' | tr -d ','`
fi
```
**Describe alternatives you've considered**

**Additional context**

**Comment by jeffpaul** [comment]
@evrpress seems like we can keep things less complex in the coding of this action if the SVN_USERNAME org or repo secret is created alongside the SVN_PASSWORD secret.  What's the use case for setting the password but not username in an org/repo secret?

**Comment by evrpress** [comment]
Well @jeffpaul, that's actually a very good point.

I started using the workflow on all of my WP plugins and add the secrets manually to each repo (which took a long time). I just recently switched to an organization where I can define this globally. 
While I understand that the password can (hopefully) not be found in the repo the username can.

So it was just for convenience to have it in place if you have many repos but it's basically useless if you use an organization.

Thank you anyway for the work on this action 💯
